modules/ROOT/pages/platform/consumption-report.adoc

```diff
@@ -0,0 +1,40 @@
+[[aura-consumption-report]]
+= Consumption report
+:description: Consumption reporting allows Virtual Dedicated Cloud customers to closely monitor their consumption.
+
+label:AuraDB-Virtual-Dedicated-Cloud[]
+
+Virtual Dedicated Cloud services are offered through prepaid consumption plans. 
+Billing is based on usage, with credits deducted from the available balance each month.
+
+The consumption report, accessible in the Aura console’s Billing section, provides real-time insights into resource usage for the current tenant, including both running and paused states. 
+It displays RAM usage in GB-hours and the equivalent cost in prepaid credits.
+
+Available to Admins, the report helps you track usage patterns over time and make informed resource allocation decisions. 
+Note that it includes primary database usage but **not** secondary database usage.
```

Comment:
note that the "secondary" external-facing name might be different.

Reply:
OK! We have a draft PR here https://github.com/neo4j/docs-aura/pull/396